glue_lab/glue_ydoc.py

```diff
@@ -41,9 +42,17 @@ def set(self, value: str) -> None:
             self._ycontents.update(t, contents.items())
             self._ytabs.update(t, tabs.items())
 
-    def observe(self, callback):
+    def observe(self, callback: Callable[[str, Any], None]):
         self.unobserve()
-        self._subscriptions[self._ystate] = self._ystate.observe(callback)
-        self._subscriptions[self._ysource] = self._ysource.observe(callback)
-        self._subscriptions[self._ycontents] = self._ycontents.observe_deep(callback)
-        self._subscriptions[self._ytabs] = self._ytabs.observe_deep(callback)
+        self._subscriptions[self._ystate] = self._ystate.observe(
+            partial(callback, "state")
```

Comment:
Do we need to do the same thing for `jupytercad`?

Reply:
Yes. I'm already on it.

Reply:
Thanks!